src/main/java/de/unistuttgart/memorybackend/controller/ConfigController.java

```diff
@@ -0,0 +1,153 @@
+package de.unistuttgart.memorybackend.controller;
+
+import de.unistuttgart.gamifyit.authentificationvalidator.JWTValidatorService;
+import de.unistuttgart.memorybackend.data.*;
+import de.unistuttgart.memorybackend.data.mapper.ConfigurationMapper;
+import de.unistuttgart.memorybackend.repositories.ConfigurationRepository;
+import de.unistuttgart.memorybackend.service.ConfigService;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.annotation.PostConstruct;
+import jakarta.validation.Valid;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@Import({ JWTValidatorService.class })
+@RestController
+@RequestMapping("/configurations")
+@Slf4j
+@Validated
+public class ConfigController {
+
+    @Autowired
+    private ConfigurationRepository configurationRepository;
+
+    @Autowired
+    private JWTValidatorService jwtValidatorService;
+
+    @Autowired
+    private ConfigService configService;
+
+    @Autowired
+    private ConfigurationMapper configurationMapper;
+
+    @PostConstruct
+    public void createDummyData() {
+        List<CardPair> pairs = Arrays.asList(
+            new CardPair(new Card("text", CardType.TEXT), new Card("text", CardType.TEXT)),
+            new CardPair(new Card("text", CardType.TEXT), new Card("text", CardType.TEXT)),
+            new CardPair(new Card("text", CardType.TEXT), new Card("text", CardType.TEXT)),
+            new CardPair(new Card("text", CardType.TEXT), new Card("text", CardType.TEXT)),
+            new CardPair(new Card("text", CardType.TEXT), new Card("text", CardType.TEXT)),
+            new CardPair(new Card("text", CardType.TEXT), new Card("text", CardType.TEXT))
+        );
+        final Configuration configuration = new Configuration(pairs);
+        Configuration savedConfig = configurationRepository.save(configuration);
+        System.out.println(savedConfig.getId());
```

Comment:
```suggestion
        log.debug("Created dummy configuration with id " + savedConfig.getId());
```
Or maybe remove the create dummy data